ironfish-rust/src/serializing/aead.rs

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is unchanged, it was just split out into it's own file, since before it was nested inside serializing.rs